Mapify/SceneInitializers/GameContent/StationSetup.cs

```diff
@@ -52,13 +52,16 @@ public override void Run()
         private static void SetupJobBookletSpawnSurface(Station station, StationController stationController)
         {
             PointOnPlane jobBookletSpawnSurface = station.transform.parent.GetComponentInChildren<PointOnPlane>();
-            if (jobBookletSpawnSurface != null)
-                return;
-            GameObject jobBookletSpawnSurfaceObject = station.gameObject.NewChildWithPosition("JobSpawnerAnchor", station.transform.TransformPoint(station.bookletSpawnArea.center));
-            jobBookletSpawnSurface = jobBookletSpawnSurfaceObject.AddComponent<PointOnPlane>();
-            Vector3 size = station.bookletSpawnArea.size;
-            jobBookletSpawnSurface.xSize = size.x;
-            jobBookletSpawnSurface.xSize = size.z;
+
+            if (jobBookletSpawnSurface == null)
+            {
+                GameObject jobBookletSpawnSurfaceObject = station.gameObject.NewChildWithPosition("JobSpawnerAnchor", station.transform.TransformPoint(station.bookletSpawnArea.center));
+                jobBookletSpawnSurface = jobBookletSpawnSurfaceObject.AddComponent<PointOnPlane>();
+                Vector3 size = station.bookletSpawnArea.size;
+                jobBookletSpawnSurface.xSize = size.x;
+                jobBookletSpawnSurface.xSize = size.z;
```

Comment:
These are both setting xSize, not sure how I ever missed that but now would be a good time to fix it

Reply:
done